File: src/problema1/python/solution.py
soma_multiplos = 0
print("Multiplos de 3 ou 5: ")

for i in range(1, 1000):
    if i % 3 == 0 or i % 5 == 0:
        print("\t" + str(i))
        soma_multiplos += i

print("A soma dos multiplos de 3 ou 5 é: " + str(soma_multiplos))

File: src/problema1/java/Solution.java
package problema1.java;

public class Solution {
    public static void main(String[] args) {
        int somaMultiplos = 0;
        System.out.println("Multiplos de 3 ou 5: ");
        for (int i = 1; i < 1000; i++) {
            if (i % 3 == 0 || i % 5 == 0) {
                System.out.println("\t" + i);
                somaMultiplos += i;
            }
        }
        System.out.println("A soma dos números multiplos de 3 ou 5: " + somaMultiplos);
    }
}
